lib/event_tracker_haystack.rb

```diff
@@ -0,0 +1,99 @@
+require 'nokogiri'
+require 'open-uri'
+require 'yaml'
+
+WEBSITE = 'http://webmontag.de'
+FACEBOOK_URL = ''
+
+def divider
+  puts "-------------------------------------------".blue
+end
+
+class String
+
+  def colorize(text, color_code)
+    "\e[#{color_code}m#{text}\e[0m"
+  end
+
+  def red
+    colorize(self, 31)
+  end
+
+  def green
+    colorize(self, 32)
+  end
+
+  def yellow
+    colorize(self, 33)
+  end
+
+  def blue
+    colorize(self, 34)
+  end
+end
+
+
+class EventBlock
+=begin
+-------------- The structured Hash -----------
+# This is how the returned hash should be
+"name"
+"facebook_url"
+"wiki_url"
+"date"
+----------
+=end
+
+    @index
+    @event_element
+    @hash
+
+  def initialize (event_element, index)
```

Comment:
I think it is okay to keep the (event_element, index) What is the point to make it individually passed?